magpie-core/src/main/java/io/openraven/magpie/core/plugins/PluginManager.java

```diff
@@ -110,7 +107,8 @@ public static Object buildPluginConfig(String pluginId, Class<?> configType, Obj
       }
     }
 
-    return MAPPER.treeToValue(MAPPER.valueToTree(config), configType);
+//    return MAPPER.treeToValue(MAPPER.valueToTree(config), configType);
```

Comment:
I need to reassess this. Was chasing a serialization bug.

Reply:
Sure thing, please let me know the end result.  I'm not 100% sure this is needed anymore.